laze-project.yml

```diff
@@ -745,6 +745,20 @@ modules:
           - -Clink-arg=-Tkeep-stack-sizes.x
           - -Z emit-stack-sizes
 
+  - name: ram-tiny
+    help: This module marks boards with very little RAM (currently, <10KiB)
+    conflicts:
```

Comment:
These are general restrictions for those small devices, at this time ...
I hope to unlock some of them in the future. E.g., a really small network stack is not unthinkable.